connector/src/test/scala/com/google/cloud/spark/bigquery/DirectBigQueryRelationSuite.scala

```diff
@@ -144,8 +149,9 @@ class DirectBigQueryRelationSuite
   }
 
   test("invalid filters with Arrow") {
-    val bigQueryRelation = new DirectBigQueryRelation(
-      SparkBigQueryOptions(ID, PROJECT_ID, readDataFormat = DataFormat.ARROW), TABLE)(sqlCtx)
+    val options = defaultOptions
+    options.readDataFormat = DataFormat.ARROW
```

Comment:
Isn't arrow the default read format?

Reply:
Yes, but the relation in the test is still configured to use Avro, see line 54